common/unsafe/src/main/java/org/apache/spark/unsafe/memory/MemoryBlock.java

```diff
@@ -22,10 +22,9 @@
 import org.apache.spark.unsafe.Platform;
 
 /**
- * A consecutive block of memory, starting at a {@link MemoryLocation} with a fixed size.
+ * A declaration of interfaces of MemoryBlock classes .
```

Comment:
`A representation of a consecutive memory block in Spark. It defines the common interfaces for memory accessing and mutating.`

Reply:
Thanks, done